src/elements/common/annotator-context/types.ts

```diff
@@ -16,13 +16,21 @@ export interface Annotator {
     removeAllListeners: () => void;
     removeListener: (event: string | symbol, listener: (...args: any[]) => void) => void;
 }
+
+export type ElementsError = {
+    code: string;
+    context_info: Record<string, unknown>;
```

Comment:
Could we use `object` instead? Would that help avoid the cast [below](https://github.com/box/box-ui-elements/pull/2172/files#diff-9f2f85578bd7824108ee496c56e3692eR111)?